File: apps/provider-api/Models/MigrationLogCurrent.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class MigrationLogCurrent
    {
        public Guid MigrationId { get; set; }
        public string ScriptChecksum { get; set; }
        public string ScriptFilename { get; set; }
        public DateTime CompleteDt { get; set; }
        public string AppliedBy { get; set; }
        public byte Deployed { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/migration-log-current.ts



export class MigrationLogCurrent {
    public migrationId: string = '';
    public scriptChecksum: string = '';
    public scriptFilename: string = '';
    public completeDt: string = '';
    public appliedBy: string = '';
    public deployed: number = 0;

    constructor(init? : Partial<MigrationLogCurrent>) {
        
        Object.assign(this, init);
    }
}

